.github/workflows/markdowner.py

```diff
@@ -213,7 +213,9 @@ def edit_codeblocks(content, filename):
 
 def perform_edits(content, flags, filename):
     if flags.trailing or flags.all:
-        replacements = {" \n": "\n", "\t\n": "\n"}
+        # Typically for markdown documents <space><space>\n adds new line without paragraph change.
```

Comment:
Please remove this commit.

Reply:
> What requirements? This is not required.
https://www.markdownguide.org/basic-syntax/#line-break-best-practices
But since it's not been followed in these docs, I've reverted the changes.